src/hugo/content/en-us/implementations/aws/command/command2/_index.md

```diff
@@ -0,0 +1,721 @@
+---
+title: "Command - Device Shadow"
+weight: 10
+summary: "Command and control of a device by using the device shadow leveraging MQTT topics"
+---
+
+Command and control is the operation of sending a message to a device requesting it to perform some action or to control the device configuration. However, when interaction happen with devices over intermittent networks or when devices deployed in far-flung areas or when using devices with limited resources, then an IoT solution may not be able to reliably send or receive commands or configuration updates. By using device shadow, IoT applications or services can simulate and control edge device configuration and execute actions.
```

Comment:
punctuation: commas between list. e.g., "intermittent networks, or when ... far-flug areas, or when using..."

Please review grammar and punctuation overall.